docs/rules/no-only-test.md

```diff
@@ -0,0 +1,35 @@
+# Ensure no `test.only()` are present
+
+It's easy to run only one test with `test.only()` and then forget about it. It's visible in the results, but still easily missed. It is potentially disastrous when ran in a CI system as mosts tests will not be run without failing the test suite. Forgetting to remove `.only`, means only this one test in the whole file will run, and if not caught, can let serious bugs slip into your codebase.
```

Comment:
I think you can remove the "It is potentially disastrous when ran in a CI system as mosts tests will not be run without failing the test suite." sentence now that we have the other one.